esp/esdllib/docs/trace-content.md

```diff
@@ -0,0 +1,30 @@
+
+### trace-content
+```xml
+    <trace-content
+        class="enumerated text literal"
+        content_type="enumerated text literal"
+        label="text literal"
+        select="XPath evaluable to text"
+        skip_mask="XPath evaluable to Boolean"
+        test="XPath evaluable to Boolean"/>
+```
+
+A childless ESDL script operation used to create a trace output entry using script generated content. If a data masking engine is configured, a masker has been selected prior to this operation, and `skip_mask` is not *true*, the generated content will be masked. The masker&apos;s `maskContent` method is invoked to locate and mask any number of values within the selected text.
```

Comment:
It can be, depending on the markdown editor in use. I frequently use an online editor that does require it. I suggest tabling this pending completion of https://track.hpccsystems.com/browse/HPCC-29109.